seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/dto/IndexInfo.java

```diff
@@ -19,19 +19,25 @@
 
 import org.apache.seatunnel.connectors.seatunnel.elasticsearch.config.SinkConfig;
 
+import java.util.List;
+
 /**
  * index config by seatunnel
  */
 public class IndexInfo {
 
     private String index;
     private String type;
+    private List<String> ids;
```

Comment:
suggest make ids final if it will never been update.

Reply:
It should be a null value if not setting index_id.